features/FEATURE_LWIP/lwip-interface/lwip-eth/arch/TARGET_STM/stm32xx_emac.c

```diff
@@ -372,6 +375,35 @@ static void _eth_arch_phy_task(void *arg)
     }
 }
 
+#if defined (TARGET_NUCLEO_F767ZI)
+/**
+ * workaround for the ETH RMII bug in STM32F769 Cut1.0
+ *
+ * \param[in] netif the lwip network interface structure
+ */
+static void _rmii_watchdog(void *arg)
+{
+    while(1) {
+        /* some good packets are received */
+        if(EthHandle.Instance->MMCRGUFCR > 0) {
```

Comment:
space `if ()` between if and starting `(`

Reply:
fixed now